sample-docker.env

```diff
@@ -0,0 +1,32 @@
+MONGOHQ_URL=mongodb://mongo:27017/freecodecamp
```

Comment:
Yes, that reduces the number of potential conflicts if keys are added at a later time.